tests/kernel/ibtests_prepare.pm

```diff
@@ -40,9 +27,9 @@ sub run {
     my $packages_master = $packages . " git-core twopence-shell-client bc";
 
 
-    $self->permit_root_login if is_ipmi;
-
     $self->select_serial_terminal;
+    permit_root_ssh;
```

Comment:
i took a look to the utils.pm. I would expect `permit_root_ssh_in_sol` here instead of `permit_root_ssh`. those two looks to do quite the same in a different way. Would it make sense to delete `permit_root_ssh_in_sol`(or merge)? However that sounds like another PR/commit. I put it here as a note.

Reply:
Right, at least for now permit_root_ssh_in_sol should be fine, too. I changed that and give it a try.